inc/thorvg.h

```diff
@@ -210,6 +210,8 @@ class TVG_EXPORT Shape final : public Paint
 public:
     ~Shape();
 
+    Result duplicate(std::unique_ptr<tvg::Shape>& shape);
+    Result duplicate(Shape* from); //For CAPI binding
```

Comment:
No, conceptually, It's not allowed.

Reply:
Let me think about it for a while. I have to consider how to call first duplicate method in CAPI bindings.